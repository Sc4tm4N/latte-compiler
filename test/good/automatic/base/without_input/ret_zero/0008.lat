// print int some equations (compared to C)
int main() {
  int eq1 = (1 * 1 - 2 * 20) % 13 + 13 / 3 * 52 + 48 * 92 % 2,
    eq2 = eq1 * eq1 + 59 % 11;
  int eq3 = 3 * 3 * 3 * 3 * 3 / 3 / 3 / 3;
  printInt(eq1);
  printInt(eq2);
  printInt(eq3);
  return 0;
}