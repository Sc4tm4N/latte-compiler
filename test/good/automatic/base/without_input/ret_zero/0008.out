208
43268
9
